Cylinder Bezier Triangle Approx

ajc428 opened the pull request.

ajc428 committed:
Support for NGA2 machine learning reconstruction

ajc428 committed:
Merge branch 'master' of https://github.com/ajc428/interface-reconstruction-library
Fixing branches

ajc428 committed:
Merge remote-tracking branch 'upstream/variant' into variant

ajc428 committed:
Cylinder Bezier Triangle Support

ajc428 committed:
Minor updates and comments added

ajc428 committed:
c interface cylinder bezier surface

ajc428 committed:
Making code consistent

fabienevrard reviewed (changes requested):
Hi Andrew,
Looks great! Before we merge, can I ask you to add 1 or 2 basic tests of the new features in `tests/src/bezier_io.cpp`?
Thanks a lot

ajc428 committed:
Cylinder Bezier Tests

ajc428 commented:
Sorry for the delay, I've added those tests.

fabienevrard committed:
Update bezier_io.cpp

Changing names of output files to avoid overwriting tests 1 and 2 when running all tests at once

fabienevrard commented:
Merged! Thanks again @ajc428
